workers/generic-worker/payload_test.go

```diff
@@ -148,6 +148,7 @@ func NowMillis(t *testing.T) (now time.Time) {
 
 // If an artifact expires before task deadline we should get a Malformed Payload
 func TestArtifactExpiresBeforeDeadline(t *testing.T) {
+	setup(t)
```

Comment:
Do you remember why this was needed, just out of interest?

Reply:
Yes! Good question. When I pulled these variables out into a different block https://github.com/taskcluster/taskcluster/pull/7418/files#diff-4d9f4299686b08adfe1957964bfc775633d85db82da2a5d71733bb3f41bcdefaR631-R632 I was getting nil pointer exceptions because the config was never set. I'm not sure why `setup(t)` was not called for these tests, but they are now, and things still look good 👍🏻

Reply:
I think because these were more like unit tests rather than integration tests, i.e. they didn't require a full system setup. But maybe now they do, for some reason.